syft/workers/base.py

```diff
@@ -327,7 +324,7 @@ def send(
 
         worker = self.get_worker(worker)
 
-        if hasattr(obj, "create_pointer"):
+        if hasattr(obj, "create_pointer"):  # TODO: this seems like hack to check a type
```

Comment:
I confirm that this check was introduced to remove a set of type checks. There are currently two classes implementing the create_pointer method. This is more the philosophy of duck typing than an explicit type check.

Reply:
Still a valid TODO. I noticed it but it's outside the scope of this PR to fix.

Reply:
So what is the todo? Is it to re-introduce a type check (would vote against) or what shall be changed?

Reply:
Seems like there must be a better way than to check the existence of a method.

Reply:
Not a high priority.